src/app/service/survery-result.service.ts

```diff
@@ -12,11 +12,10 @@ import { Router } from '@angular/router';
 // const perf_back_api = 'http://127.0.0.1:5000/';
```

Comment:
if you need the localhost for debugging, you can set a global variable to determine this dynamically. might want to do this for test setups.

Reply:
Good idea. I'm not sure how to do that at the moment, so I'm keeping it but would like to learn/find out how to do it soon.

Reply:
```console.log``` can be replaced with a similar wrapper as well so that they don't run on production environment.